deploy/ansible/playbooks/roles/horizon-start/tasks/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
 
-- name: pull docker image
+- name: pull docker image. this tends to fail on the first time.
+  docker_service:
+    project_src: /opt/horizon
+    stopped: yes
```

Comment:
`state: absent` in other tasks that do the same thing - why? make all tasks that pull images use the same syntax